ZaeXT/test-scripts/test_failures.py

```
@@ -0,0 +1,183 @@
+    password = "password123"
+    payload = {"username": username, "password": password}
+    
+    print("  -> 第一次注册 (应该成功)")
+    res_first = requests.post(f"{BASE_URL}/register", json=payload)
+    if res_first.status_code != 200:
+        print_fail("为测试重复注册做准备时，首次注册失败", res_first)
+    
+    print("  -> 尝试用相同的用户名再次注册")
+    res_second = requests.post(f"{BASE_URL}/register", json=payload)
+    # 预期服务器会返回一个通用错误 (500) 或一个更具体的冲突错误 (409)
+    # 根据我们的 handler 实现，返回的是 500 + 错误信息
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)